lib/langAssembler.js

```diff
@@ -15,6 +15,14 @@ module.exports.assemble = assemble;
 function assemble(callback) {
     var localeDirectory = 'lang';
 
+    try {
+        Fs.statSync(localeDirectory);
+    }
+    catch (e) {
+        // No lang folder, callback immediately
+        return callback(null, 'no_lang_folder');
```

Comment:
I think you should return an error in the callback and then check the error type in `bin/stencil-bundle:294`